src/app/pages/data-protection/cloudsync/cloudsync-task-card/cloudsync-task-card.component.ts

```diff
@@ -51,6 +51,8 @@ export class CloudSyncTaskCardComponent implements OnInit {
   jobStates = new Map<number, string>();
   readonly jobState = JobState;
 
+  private ixChainedSlideInService = inject(IxChainedSlideInService);
```

Comment:
I like this approach, but it breaks component consistency. Could we move it to the constructor?